### PR TITLE
release notes for feb28th to mar6th

### PR DESCRIPTION
**Pull request by qpetraroia**

### CODE REVIEW

**Comment by palma21** [comment]
cc @Azure/aks-pm

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
...
+## Release 2021-03-08
+
+This release is rolling out to all regions - ETA for conclusion 2021-03-17 for public cloud.
+
+### Announcements
+
+* The Azure Kubernetes Service [pod security policy (preview)](https://docs.microsoft.com/azure/aks/use-pod-security-policies) feature will be retired on June 30th, 2021.
+* Once GA AKS will default to its new [GPU specialized image](https://aka.ms/aks/specialized-gpu-image) as the supported option for GPU-capable agent nodes.
+* Starting last week, the week of Feb 22nd (Azure China Cloud and Azure Government Cloud users will get this update in the following weeks), we will upgrade AKS clusters Calico network policy from Calico version v3.8.9 to v3.17.2 for cluster 1.20.2 and above. This upgrade will cause a breaking change to the default behavior of all-interfaces Host Endpoints. For customers that use Host Endpoints, and only these, this version brings a change. Please follow our [guidance](https://github.com/Azure/AKS/issues/2089) to apply the appropriate label and Global Network Policy if you want to keep the v3.8.9 default behavior of all-interfaces Host Endpoints.
+* Kubernetes version 1.17 will be deprecated at the end of March 2021.
```
Comment (palma21) [inline CHANGELOG.md:12]: Let's go with the specific week?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
 
+## Release 2021-03-08
+
+This release is rolling out to all regions - ETA for conclusion 2021-03-17 for public cloud.
+
+### Announcements
+
+* The Azure Kubernetes Service [pod security policy (preview)](https://docs.microsoft.com/azure/aks/use-pod-security-policies) feature will be retired on June 30th, 2021.
+* Once GA AKS will default to its new [GPU specialized image](https://aka.ms/aks/specialized-gpu-image) as the supported option for GPU-capable agent nodes.
+* Starting last week, the week of Feb 22nd (Azure China Cloud and Azure Government Cloud users will get this update in the following weeks), we will upgrade AKS clusters Calico network policy from Calico version v3.8.9 to v3.17.2 for cluster 1.20.2 and above. This upgrade will cause a breaking change to the default behavior of all-interfaces Host Endpoints. For customers that use Host Endpoints, and only these, this version brings a change. Please follow our [guidance](https://github.com/Azure/AKS/issues/2089) to apply the appropriate label and Global Network Policy if you want to keep the v3.8.9 default behavior of all-interfaces Host Endpoints.
```
Comment (palma21) [inline CHANGELOG.md:11]: I Think we can remove this one since it was pre-announced and announced already?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,35 @@
...
+* Kubernetes version 1.17 will be deprecated at the end of March 2021.
+
+### Release Notes
+
+* Features
+  * Azure monitor for containers now supports Pods & Replica set live logs in AKS resource view. Read more [here](https://azure.microsoft.com/updates/azmon-livelogs-pods/)
+  * Confidential Compute Addon SGX Device plugin daemonset now has a seamless integration with Intel Icelake architecture and Intel's open source device plugin implementation.
```
Comment (palma21) [inline CHANGELOG.md:18]: Pending validation from @agowdamsft

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,35 @@
...
+  * Kubenet support for Pod Identity.
```
Comment (palma21) [inline CHANGELOG.md:19]: This should be Preview Feature

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,35 @@
...
+* Bug Fixes
+  * The latest Windows image fixes a bug where Windows would randomly break nodes at the CNI level and cause all pods scheduled on that node to be permanently stuck, or blocked during deployment. If you have questions about this fix, please contact the [Windows Container Team](https://github.com/microsoft/Windows-Containers).
```
Comment (palma21) [inline CHANGELOG.md:21]: would randomly break -> could break

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,35 @@
...
+  * Fix duplicate packet for kubenet on containerd when traffic is sent to another pod on the same node over cluster service IP.
+  * Fixed bug in addon_profile that caused crashes on build using Terraform in [sov clouds](https://github.com/terraform-providers/terraform-provider-azurerm/issues/6462).
```
Comment (palma21) [inline CHANGELOG.md:23]: Addon profile API

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,35 @@
...
+
+### Release Notes
+
+* Features
+  * Azure monitor for containers now supports Pods & Replica set live logs in AKS resource view. Read more [here](https://azure.microsoft.com/updates/azmon-livelogs-pods/)
+  * Confidential Compute Addon SGX Device plugin daemonset now has a seamless integration with Intel Icelake architecture and Intel's open source device plugin implementation.
+  * Kubenet support for Pod Identity.
+* Bug Fixes
+  * The latest Windows image fixes a bug where Windows would randomly break nodes at the CNI level and cause all pods scheduled on that node to be permanently stuck, or blocked during deployment. If you have questions about this fix, please contact the [Windows Container Team](https://github.com/microsoft/Windows-Containers).
+  * Fix duplicate packet for kubenet on containerd when traffic is sent to another pod on the same node over cluster service IP.
```
Comment (palma21) [inline CHANGELOG.md:22]: No issues about this on GH to link? Thought there was one

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,35 @@
...
+  * Fixed bug in addon_profile that caused crashes on build using Terraform in [sov clouds](https://github.com/terraform-providers/terraform-provider-azurerm/issues/6462).
+* Behavioral Change
+  * The Pod Identity count limitation has been raised from 50 to 200.
```
Comment (palma21) [inline CHANGELOG.md:25]: The maximum number of managed identities for the Pod Identity addon was increased from 50 to 200.
@miwithro has the doc been updated as well?

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,35 @@
...
+  * Changed default behavior of Node Pool logging to only log the error, but not return any error.
```
Comment (palma21) [inline CHANGELOG.md:26]: This does not seem external, follow up offline

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,35 @@
...
+  * Systemd-resolved will no longer be used in AKS Ubuntu 18.04 images. This resolves past issues regarding private DNS with .local entries not working with [Kubernetes 1.18 and Ubuntu 18.04](https://github.com/Azure/AKS/issues/2052).
```
Comment (palma21) [inline CHANGELOG.md:27]: Provide this week's image version as the one where this chance starts from

**Review by palma21 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (qpetraroia) [reply CHANGELOG.md:11]: yep talked to keiko. Taking it out now

File: CHANGELOG.md (hunk 8 above)
Comment (miwithro) [reply CHANGELOG.md:25]: Yes already posted.

https://docs.microsoft.com/en-us/azure/aks/use-azure-ad-pod-identity

Comment (qpetraroia) [reply CHANGELOG.md:25]: @palma21 I can take care of this if it hasn't been done

**Review by weinong — APPROVED** [review]

**Review by keikhara — APPROVED** [review]

**Review by palma21 — APPROVED** [review]

**Pull request merged**